src/resources/postcss/base/full/forms/_checkboxes-radios.pcss

```diff
@@ -66,13 +66,19 @@
 	.tribe-common-form-control-checkbox,
 	.tribe-common-form-control-radio {
 		line-height: 0;
+
+		&:hover {
+			opacity: var(--opacity-icon-hover);
+		}
```

Comment:
i believe you'd get a double-hit on opacity with this, no? if you hover over the label, what happens?